### ISSUE TITLE
v1.3.3: config param "debug": true does not enable debug logging

### ISSUE THREAD

**Issue by dutchiechris** [NetApp/netappdvp#69]
With v1.3.3 setting to `debug` to `true` in the config json appears to have no impact on the message level logged (vs. 1.3 where it did).

Config json syntax:
```
{
    "version": 1,
    "debug": true,
    "storageDriverName": "ontap-nas",
    "managementLIF": "10.64.28.91",
    "svm": "docker-ontap",
    "username": "vsadmin",
    "password": "not-my-real-password",
    "aggregate": "n1_sata",
    "storagePrefix": "docker7test_"
}
```

Create a volume and this is logged:

v1.3 (first part, it is long...):
```
DEBU[265066] Getting path for volume 'docker5_chris_test' as '/var/lib/docker-volumes/ontap-nas/docker5_chris_test'
DEBU[265066] Create({chris_test map[]})
DEBU[265066] OntapNASStorageDriver#Create(docker5_chris_test)
DEBU[265066] sending to '10.64.28.91' xml:
<?xml version="1.0" encoding="UTF-8"?>
        <netapp xmlns="http://www.netapp.com/filer/admin" version="1.21" vfiler="docker-ontap">
             <volume-size>
     <volume>docker5_chris_test</volume>
 </volume-size>
        </netapp>
DEBU[265066] URL:> https://10.64.28.91/servlets/netapp.servlets.admin.XMLrequest_filer
DEBU[265066] response Status: 200 OK

```

v1.3.3:
```
3-docker7 etc # cat /var/log/netappdvp/ontap-nas-test.log
INFO[2017-02-06T11:27:59Z] Using storage driver: ontap-nas
INFO[2017-02-06T11:27:59Z] Using config: {1 ontap-nas true false [34 100 111 99 107 101 114 55 116 101 115 116 95 34] []}
INFO[2017-02-06T11:27:59Z] Starting docker volume plugin with the following options:  port=27611 volume-driver=ontap-nas-test volumeDir="/var/lib/docker-volumes/ontap-nas-test"

```

My SolidFire config seems to have the same issue with v1.3.3 so I presume the log check for debug true/false broke in some common code.

**Comment by dutchiechris** [comment]
I noticed that if you pass the `-debug` flag to the netappdvp binary when starting it you ***do*** get debug mode enabled.  So there is a workaround...

**Comment by adkerr** [comment]
Internally tracked by [DE284](https://rally1.rallydev.com/#/47479883219d/detail/defect/91328387688)

**Comment by innergy** [comment]
Fixed in 17.04.0
